app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,120 @@
+//Todo
+// 5. build. gradel Tpfvm flqb
+
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.models.Task;
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.HttpClientErrorException;
+
+
+@RestController
+@RequestMapping("/tasks")
+@CrossOrigin
+public class TaskController {
+
+  private Long id = 0L;
+  private HashMap<Long, Task> tasks = new HashMap();
+
+  /**
+   * http get 요청을 받아 알일들을 리턴합니다.
```

Comment:
오타가 있군요. javadoc은 정적 사이트로 만들어 다른 사람들에게 보여줄 것이기 때문에 문장에 신경을 많이 써야 합니다.
```suggestion
   * http get 요청을 받아 할 일들을 리턴합니다.
```